### PR TITLE
MSFragger remove unused parameter

### PR DESCRIPTION
**Pull request by timosachsenberg**
# Description

Please include a summary of the change and which issue is fixed.

# Checklist:
- [ ] Make sure that you are listed in the AUTHORS file
- [ ] Add relevant changes and new features to the CHANGELOG file
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] New and existing unit tests pass locally with my changes
- [ ] Updated or added python bindings for changed or new classes. (Tick if no updates were necessary.)

# How can I get additional information on failed tests during CI:
If your PR is failing you can check out 
- http://cdash.openms.de/index.php?project=OpenMS and look for your PR. If you click in the column that lists the failed tests you will get detailed error messages.

# Note:
- Once you opened a PR try to minimize the number of *pushes* to it as every push will trigger CI (automated builds and test) and is rather heavy on our infrastructure (e.g., if several pushes per day are performed).

### CODE REVIEW

File: src/openms/source/ANALYSIS/ID/PercolatorFeatureSetHelper.cpp [hunk 1]
```diff
@@ -163,7 +163,7 @@ namespace OpenMS
       }
     }
 
-    void PercolatorFeatureSetHelper::addMSFRAGGERFeatures(vector<PeptideIdentification>& peptide_ids, StringList& feature_set)
+    void PercolatorFeatureSetHelper::addMSFRAGGERFeatures(StringList& feature_set)
```
Comment (oliveralka) [inline src/openms/source/ANALYSIS/ID/PercolatorFeatureSetHelper.cpp:166]: use "extra_features" as a variable name and change the ones below?

File: src/utils/PSMFeatureExtractor.cpp [hunk 2]
```diff
@@ -257,11 +257,26 @@ class PSMFeatureExtractor :
...
+    {
+      PercolatorFeatureSetHelper::addXTANDEMFeatures(all_peptide_ids, feature_set);
+    }
+    else if (search_engine == "Comet") 
+    {
+      PercolatorFeatureSetHelper::addCOMETFeatures(all_peptide_ids, feature_set);
+    }
+    else if (search_engine == "MSFragger") 
+    {
+      PercolatorFeatureSetHelper::addMSFRAGGERFeatures(feature_set);
```
Comment (oliveralka) [inline src/utils/PSMFeatureExtractor.cpp:278]: similar to the variable name above?

**Review by oliveralka — CHANGES_REQUESTED** [review]
Looks good, two minor commets.

File: src/openms/source/ANALYSIS/ID/PercolatorFeatureSetHelper.cpp (hunk 1 above)
Comment (timosachsenberg) [reply src/openms/source/ANALYSIS/ID/PercolatorFeatureSetHelper.cpp:166]: these are not extra features like in the other case but the ones directly returned by MSFragger

**Pull request merged**